docs/howto/without_active_record.md

```diff
@@ -0,0 +1,116 @@
+# How to use AMS without ActiveRecord
+
+## Defining Models With ActiveModel
+
+When using active model, there are only a few methods you'll need to implement.
+* attributes
+* read_attribute_for_serialization
+* cache_key
+* id
+
+Here is an example base model that could be used to extend all your models with
+
+```ruby
+require 'active_model'
+
+class BaseModel
+  include ActiveModel::Model
+  include ActiveModel::Serialization
+
+  attr_accessor :id, :updated_at
+
+  def initialize(hash = {})
```

Comment:
Understood. :-)
I'll rework this when I get back from dinner

On Mon, Oct 5, 2015, 5:52 PM Lucas Hosseini <notifications@github.com>
wrote:

> In docs/howto/without_active_record.md
> <https://github.com/rails-api/active_model_serializers/pull/1241#discussion_r41202712>
> :
>
> > +* read_attribute_for_serialization
> > +* cache_key
> > +* id
> > +
> > +Here is an example base model that could be used to extend all your models with
> > +
> > +```ruby
> > +require 'active_model'
> > +
> > +class BaseModel
> > +  include ActiveModel::Model
> > +  include ActiveModel::Serialization
> > +
> > +  attr_accessor :id, :updated_at
> > +
> > +  def initialize(hash = {})
>
> What I meant is that putting this in the docs is misleading because it
> makes people believe AMS needs this constructor to work.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails-api/active_model_serializers/pull/1241/files#r41202712>
> .
>